.github/release.yml

```diff
@@ -0,0 +1,23 @@
+# Release notes template (https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes)
+
+changelog:
+  categories:
+    - title: Breaking Changes 🚧
+      labels:
+        # Semver-Major
+        - breaking change
```

Comment:
You might opt to include https://github.com/dchbx/resource_registry/blob/main/.github/workflows/label-checker.yml in this PR. The GH release notes will group PRs merged, based on labels, as defined in this release.yml. Enforcing use of the expected set could help with consistency. 

(Might also consider adding the possible labels to this project - the DC repos should all have them, with some `*` denotation to show labels that satisfy the checker. E.g, https://github.com/dchbx/resource_registry/issues/labels).